pkg/config/config_template.yaml

```diff
@@ -6,10 +6,16 @@
 ## @param api_key - string - required
 ## @env DD_API_KEY - string - required
 ## The Datadog API key to associate your Agent's data with your organization.
-## Create a new API key here: https://app.datadoghq.com/account/settings
+## Create a new API key here: https://app.datadoghq.com/organization-settings/api-keys
 #
 api_key:
 
+## @param app_key - string - optional
+## @env DD_APP_KEY - string - optional
+## The Datadog application key to use when your Agent makes requests to Datadog.
```

Comment:
@romainkomorndatadog I think the two descriptions need maybe a bit more detail to prevent some users from confusing the intent of the two fields. Effectively neither really describes very well how they're different from each other outside of clicking the links and even those pages don't have good descriptions.

Reply:
@sgnn7 , I updated the comment strings to reflect the wording in our docs website: https://docs.datadoghq.com/account_management/api-app-keys/ .